markers-custom/README.md

```diff
@@ -0,0 +1,18 @@
+Basic example
+---
+
+Initialize a map with coordinates set by React, update the state of coordinates on map movement, and display that data in an element provided by the render function.
+
+![Demo gif](https://i.imgur.com/IGJuTuM.gif)
+
+### Run it
+
+    git clone https://github.com/mapbox/mapbox-react-examples.git
+
+    cd mapbox-react-examples/basic
+    npm install
+    npm start
+
+    open http://localhost:3000/
+
+Runs the app from http://localhost:3000
```

Comment:
Can you update the READMEs of the three examples to reflect what they do? No need to include a visual example and these can be pretty short.